### PR TITLE
Background Task Runner example

### PR DESCRIPTION
**Pull request by warrenbuckley**
This includes a working example of regularly emptying out the recycle bin

### CODE REVIEW

File: Reference/Scheduling/index.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
+---
+versionFrom: 8.0.0
+---
+
+# Scheduling with BackgroundTaskRunner
+In version Umbraco 8+ it is possible to run re-occuring code using the `BackgroundTaskRunner`.
```
Comment (sofietoft) [inline Reference/Scheduling/index.md]:
```suggestion
In Umbraco 8+ it is possible to run reccuring code using the `BackgroundTaskRunner`.
```

File: Reference/Scheduling/index.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+Below is a complete example showing how to register the Task with the runner with a component that will regularly empty out the recycle bin every five minutes.
```
Comment (sofietoft) [inline Reference/Scheduling/index.md]:
```suggestion
Below is a complete example showing how to register a Task with the runner with a component that will regularly empty out the recycle bin every five minutes.
```
Will it empty out the recycle bin every five minutes, or regularly? 🤔

Comment (warrenbuckley) [reply Reference/Scheduling/index.md]: Yes every five minutes, I know a bit over the top for an example but gives people some inspiration/ideas on what they could do hopefully @sofietoft

File: Reference/Scheduling/index.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
+---
+versionFrom: 8.0.0
+---
+
+# Scheduling with BackgroundTaskRunner
+In Umbraco 8+ it is possible to run reccuring code using the `BackgroundTaskRunner`.
+Below is a complete example showing how to register a Task with the runner with a component that will regularly empty out the recycle bin every five minutes.
```
Comment (jmayntzhusen) [inline Reference/Scheduling/index.md]:
```suggestion
Below is a complete example showing how to register a Task Runner with a [component](../../Implementation/Composing/index.md) that will regularly empty out the recycle bin every five minutes.
```

File: Reference/Scheduling/index.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+```csharp
+using Umbraco.Core;
+using Umbraco.Core.Composing;
+using Umbraco.Core.Logging;
+using Umbraco.Core.Services;
+using Umbraco.Web.Scheduling;
+
+namespace Umbraco.Web.UI
+{
+    public class CleanUpYourRoomComposer : ComponentComposer<CleanUpYourRoomComponent>
```
Comment (jmayntzhusen) [inline Reference/Scheduling/index.md]:
```suggestion
    // We start by setting up a composer and component so our task runner gets registered on application startup
    public class CleanUpYourRoomComposer : ComponentComposer<CleanUpYourRoomComponent>
```

File: Reference/Scheduling/index.md [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+            //As soon as we add our task to the runner it will start to run (after its delay period)
+            _cleanUpYourRoomRunner.TryAdd(task);
+        }
+
+        public void Terminate()
+        {
+        }
+    }
+
+    public class CleanRoom : RecurringTaskBase
```
Comment (jmayntzhusen) [inline Reference/Scheduling/index.md]:
```suggestion
    // Now we get to define the recurring task
    public class CleanRoom : RecurringTaskBase
```

**Pull request merged**